modeshape-jcr/src/main/java/org/modeshape/jcr/cache/document/DocumentConstants.java

```diff
@@ -46,6 +46,9 @@
     public static final String REFERENCE_FIELD = "$ref";
     public static final String WEAK_REFERENCE_FIELD = "$wref";
     public static final String SIMPLE_REFERENCE_FIELD = "$sref";
+    public static final String PRB = "$prb";
```

Comment:
BTW, it's okay to use something short as the literal. If you want to keep using `PRB`, then perhaps add a comment.